CHANGES.md

```diff
@@ -5,6 +5,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 ### Notable changes
 
+* Bumped DC/OS UI to [1.13+v2.82.3](https://github.com/dcos/dcos-ui/releases/tag/1.13+v2.82.3)
```

Comment:
If this is created by automated tooling, can we change the wording to "Updated to ..." @juliangieseke?

Reply:
yeah, it kinda is, will change wording 👍

Reply:
Thank you!